### PR TITLE
Enable maintenance

### PR DESCRIPTION
**Pull request by bobo-k2**
**Pull Request Summary**

> Describe what changes this pull request makes to the repository

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

**This pull request makes the following changes:**

**Adds**
- (ex: Add feature A)

**Fixes**
- (ex: Fix validation function)

**Changes**
- (ex: Change document B)

**To-dos**
> *Feel free to remove this section if it's not applicable

- [ ] (ex: add user list)

### CODE REVIEW

**Review by impelcrypto — APPROVED** [review]

File: src/store/dapp-staking/actions.ts [hunk 1]
```diff
@@ -798,15 +798,15 @@ const actions: ActionTree<State, StateInterface> = {
...
-        commit('setIsPalletDisabled', isPalletDisabled);
+        // let isPalletDisabled = false;
+        // try {
+        //   const isDisabled = await $api.value.query.dappsStaking.palletDisabled<bool>();
+        //   isPalletDisabled = isDisabled.valueOf();
+        // } catch {
+        //   // palletDisabled storage item is not supported by a node;
+        // }
+
+        commit('setIsPalletDisabled', true);
```
Comment (hoonsubin) [inline src/store/dapp-staking/actions.ts:809]: Can you make it so that it can allow Shiden users to continue using dapps staking while only filter astar?

**Review by hoonsubin — CHANGES_REQUESTED** [review]

**Review by fiexer — APPROVED** [review]

File: src/store/dapp-staking/actions.ts (hunk 1 above)
Comment (impelcrypto) [reply src/store/dapp-staking/actions.ts:809]: @bobo-k2 
I see there is no `palletDisabled` query in Astar. How about:
```
// Memo: "Astar" or allocate node version like `isIndividualClaim`
if(network === Astar){
  return true
}else{
  const isDisabled = await $api.value.query.dappsStaking.palletDisabled<bool>();
  return isDisabled.valueOf();
}
```

Comment (hoonsubin) [reply src/store/dapp-staking/actions.ts:809]: Sure, that works

Comment (bobo-k2) [reply src/store/dapp-staking/actions.ts]: I made a quick fix. I will merge now and polish later

Comment (bobo-k2) [reply src/store/dapp-staking/actions.ts]: Added some improvements

**Review by hoonsubin — APPROVED** [review]
LGTM

**Pull request merged**